docs/content/v1.0.x/release-notes/_index.md

```diff
@@ -5,6 +5,13 @@ menu:
 docs:
 weight: 100
 ---
+sectionStart
+### v1.0.10
+
+Add handling when using invalid ArbitraryIntrospector. Logs when container type, throws an exception if the concrete type.
```

Comment:
A detailed release-note might be better for the users!

Add exception handling when using an `ArbitraryIntrospector` that does not match the property type. Add a log entry if the property is of a container type, and throw an exception if the property is of a concrete type.